validators/authenticate.js

```diff
@@ -0,0 +1,43 @@
+const passport = require("passport");
+const passportJWT = require("passport-jwt");
+const User = require("../models/userSchema");
+require("dotenv").config();
+const secret = process.env.SECRET;
+
+const ExtractJWT = passportJWT.ExtractJwt;
+const Strategy = passportJWT.Strategy;
+const params = {
+  secretOrKey: secret,
+  jwtFromRequest: ExtractJWT.fromAuthHeaderAsBearerToken(),
+};
+
+// JWT Strategy
+passport.use(
+  new Strategy(params, function (payload, done) {
+    User.find({ _id: payload.id })
+      .then(([user]) => {
```

Comment:
Trzymaj się jednej składni raczej, czyli jak piszesz z async await to zostań z tym i tutaj.